### DISCUSSION TITLE
Timelines to add headers to a release

### DISCUSSION THREAD

**Discussion by rupesh3020** [AbsaOSS/spline-spark-agent#523] [General]
Hi Team,

When can we expect the new version of jar which will contain headers feature ? 
7.12 was released but i don't see it supporting HTTP headers to be specified in configuration.

**Comment by wajda** [comment]
It comes with 1.0.0. Should be released soon, but not later than the end of November. We are finalizing changes across all Spline sub-projects. Meantime I encourage you to give the latest 1.0.0 build a test and let us know your feedback.
https://teamcity.jetbrains.com/project/OpenSourceProjects_AbsaOSS_SplineAgentSpark?branch=%3Cdefault%3E&mode=builds

**Reply by ganeshnikumbh** [reply]
@wajda can we expect the latest jar by 30th November? or will it take some more time? We are really looking forward to the header feature.

**Reply by wajda** [reply]
no, sorry, the month have passed way too quickly

**Reply by ganeshnikumbh** [reply]
@wajda thanks for your reply. Is it possible to give a tentative date for this release?

**Reply by wajda** [reply]
Maybe a month, but don't take my estimations seriously. It all depends on circumstances. But we are working to make it happen, so please bear with us.

**Reply by rupesh3020** [reply]
Hi @wajda ,

We are also using spline in one of our very crucial projects and are waiting for the features coming in version 1.0.0.  We have high hopes for this product as this solves some very specific problems that other tools are not able to solve currently. This is the reason we are also recommending this to other clients but it depends on the success of the current implementation. Can you please check if this can be released before 15th of this month? We are also looking to move to other tools but I am sure then we will miss the simple and flexible nature of spline.

**Reply by wajda** [reply]
We'll do what we can. We are really close to release, just want to consolidate changes in different modules.

**Reply by wajda** [reply]
there :) - https://github.com/AbsaOSS/spline-spark-agent/releases/tag/release%2F1.0.0

**Reply by ganeshnikumbh** [reply]
Thank you very much @wajda :)
